build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import com.vanniktech.maven.publish.MavenPublishPluginExtension
+import com.vanniktech.maven.publish.SonatypeHost
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+        gradlePluginPortal()
+        maven(url = "https://maven.pkg.jetbrains.space/public/p/compose/dev" )
+    }
+
+    dependencies {
+        classpath(libs.plugin.hilt)
+        classpath(libs.plugin.ktlint)
+        classpath(libs.plugin.maven)
+        classpath(libs.plugin.multiplatform.compose)

```

Comment:
It's possible to use the bundle to do something like `libs.bundles.plugins.forEach(::classpath)`?